auth-web/src/locales/en.json

```diff
@@ -76,7 +76,7 @@
   "passwordResetFailureText": "An error occurred while attempting to reset this password.\nPlease try again later.",
   "tos_updated": "We have updated our Terms of Use. Please review and accept the new terms to proceed.",
   "createBCRegistriesAccount": "Create a BC Registries Account",
-  "pendingAffidavitReviewTitle": "Thanks for creating the account {team} at BC Registries & Online Services.",
+  "pendingAffidavitReviewTitle": "Thanks for creating the account {team} at BC Registries and Online Services.",
   "pendingAffidvitReviewMessage": "Once your account has been reviewed and approved you will receive\n\nan email allowing you to activate and manage the account. \n\nPlease allow 5 business days for this process.",
   "extraProdOrgSuccessTitle": "Your BC Registries account has been created and is being reviewed.",
```

Comment:
"Your BC Registries and Online Services account has been created and is being reviewed."